src/ui/categories/common/CategoryTreeSelectorRow.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import { Button, Label } from 'patternfly-react';
+
+import TreeNodeFolderIcon from 'ui/common/tree-node/TreeNodeFolderIcon';
+import TreeNodeExpandedIcon from 'ui/common/tree-node/TreeNodeExpandedIcon';
+import RowSpinner from 'ui/common/RowSpinner';
+
+class CategoryTreeSelectorRow extends Component {
```

Comment:
this is indeed meant to be rendering a row and not rows.
if we want to keep it that way it should either be named rows or the mapping should be happening before consuming the actual component (which is probably what I'd go for).